pyface/ui/qt4/code_editor/code_widget.py

```diff
@@ -31,7 +31,7 @@ class CodeWidget(QtGui.QPlainTextEdit):
     focus_lost = QtCore.Signal()
 
     def __init__(
-        self, parent, should_highlight_current_line=True, font=None, lexer=None
+        self, parent, should_highlight_current_line=False, font=None, lexer=None
```

Comment:
Just FYI - the code line highlight no longer shows up when running the `examples/python_editor/python_editor_application.py` app. I presume this is since the Qt5 `AdvancedCodeWidget` class uses the default value of `should_highlight_current_line` when instantiating the `AdvancedCodeWidget.code` attribute:

https://github.com/enthought/pyface/blob/8ad15a8a64f095c48c017575bc97772547b321e5/pyface/ui/qt4/code_editor/code_widget.py#L563

Is this intentional / expected?

Reply:
Ah, I guess from 
> This also changes the default for the code editor to not highlight the current line, as that color should probably have some thought put into it if it is used.
It would be expected?

Reply:
Those whole set of classes need a refactor, because a lot of stuff doesn't surface up to the `PythonEditor` widget.

Some of this is captured as an item here: https://github.com/enthought/pyface/issues/551

I'll add a more specific issue.

Reply:
Added #1094